pinot-clients/pinot-jdbc-client/src/test/java/org/apache/pinot/client/PinotDriverTest.java

```diff
@@ -66,6 +68,7 @@ public void testDriver()
   public void testDriverBadURL() {
     try {
       DriverManager.getConnection(BAD_URL);
+      fail("expected exception was not thrown");
```

Comment:
Suggest using `expectedExceptions` and `expectedExceptionsMessageRegExp` in the `@Test` annotation

Reply:
Done.